draft-ietf-masque-connect-ip-dns.md

```diff
@@ -398,23 +361,16 @@ could break the privacy properties provided by the tunnel. To avoid this,
 implementations need to ensure that DNS_ASSIGN capsules are not sent before the
 corresponding ROUTE_ADVERTISEMENT capsule.
 
-The requirement for an endpoint to always send DNS_ASSIGN capsules in response
-to DNS_REQUEST capsules could lead it to buffer unbounded amounts of memory if
-the underlying stream is blocked by flow or congestion control. Implementations
-MUST place an upper bound on that buffering and abort the stream if that limit
-is reached.
-
 # IANA Considerations {#iana}
 
-This document, if approved, will request IANA add the following values to the
+This document, if approved, will request IANA add the following value to the
 "HTTP Capsule Types" registry maintained at
 <[](https://www.iana.org/assignments/masque)>.
 
 |   Value   | Capsule Type |
 |:----------|:-------------|
 | 0x818F79E |  DNS_ASSIGN  |
-| 0x818F79F |  DNS_REQUEST |
-{: #iana-capsules-table title="New Capsules"}
+{: #iana-capsules-table title="New Capsule"}
```

Comment:
We need to change the value since we changed the contents. Use this:

https://martinthomson.github.io/quic-pick/#seed=draft-ietf-masque-connect-ip-dns-04;field=capsule;codepoint=0x1ace79ec;size=4